strings/phobia.json

```diff
@@ -213,5 +213,59 @@
 		"as (unknown)",
 		"unknown",
 		"stranger danger"
+	],
+
+"birds": [
+		"when are we adding vox",
+		"when you send me the sprites",
+		"or just link me to whichever server repo has them",
+		"i will port them right now",
```

Comment:
We will never add vox. I find these four lines a big injoke though.

Reply:
Not my words.

Couldn't be a player reference either, that'd be wrong.

Reply:
add `kiki` and `yaya`